src/results/financial.jl

```diff
@@ -146,7 +146,8 @@ function initial_capex(m::JuMP.AbstractModel, p::REoptInputs; _n="")
     for b in p.s.storage.types.elec
         if p.s.storage.attr[b].max_kw > 0
             initial_capex += p.s.storage.attr[b].installed_cost_per_kw * value.(m[Symbol("dvStoragePower"*_n)])[b] + 
-                p.s.storage.attr[b].installed_cost_per_kwh * value.(m[Symbol("dvStorageEnergy"*_n)])[b]
+                p.s.storage.attr[b].installed_cost_per_kwh * value.(m[Symbol("dvStorageEnergy"*_n)])[b] +
+                (p.s.storage.attr[b].installed_cost_constant * ceil(value.(m[Symbol("dvStorageEnergy"*_n)])[b]/(value.(m[Symbol("dvStorageEnergy"*_n)])[b] + 1)))
```

Comment:
@toddleif why is the `* ceil...` multiplier necessary here?

Reply:
That's a great idea. I just switched it to binIncludeStorageCostConstant.